tabular/src/autogluon/tabular/models/rf/rf_model.py

```diff
@@ -27,12 +25,25 @@ class RFModel(AbstractModel):
     def __init__(self, **kwargs):
         super().__init__(**kwargs)
         self._feature_generator = None
+        self._daal = False  # Whether daal4py backend is being used
 
     def _get_model_type(self):
+        if self.problem_type == QUANTILE:
+            from .rf_quantile import RandomForestQuantileRegressor
+            return RandomForestQuantileRegressor
+        if self.params_aux.get('use_daal', False):
+            # Disabled by default because it appears to degrade performance
+            try:
+                from daal4py.sklearn.ensemble import RandomForestClassifier, RandomForestRegressor
+                self._daal = True
+            except:
+                from sklearn.ensemble import RandomForestClassifier, RandomForestRegressor
```

Comment:
maybe print tip to install daal4py in order to get faster RF models?

Reply:
daal4py actually slowed RF models down from the tests I've ran. This is why its disabled by default until more testing is done.